js/app.js

```diff
@@ -0,0 +1,138 @@
+'use strict'
+
+var totalGuesses = 25;
+
+// alert("Can you see this ?");
+// debugger;
+
+/*
+// STRETCH GOAL
+// https://www.w3schools.com/howto/howto_js_tabs.asp
+
+// ADD NAV TABS AND FUNCTIONALITY FOR THOSE TABS
+function openPage(evt, pageName) {
+  // Declare all variables
+  var i, tabcontent, tablinks;
+
+  // Get all elements with class="tabcontent" and hide them
+  tabcontent = document.getElementsByClassName("tabcontent");
+  for (i = 0; i < tabcontent.length; i++) {
+      tabcontent[i].style.display = "none";
+  }
+
+  // Get all elements with class="tablinks" and remove the class "active"
+  tablinks = document.getElementsByClassName("tablinks");
+  for (i = 0; i < tablinks.length; i++) {
+      tablinks[i].className = tablinks[i].className.replace(" active", "");
+  }
+
+  // Show the current tab, and add an "active" class to the button that opened the tab
+  document.getElementById(pageName).style.display = "block";
+  evt.currentTarget.className += " active";
+}
+*/
+
+// ARRAY TO HOLD ALL PRODUCT OBJECTS
+Product.allProducts = [];
+
+// MAKE A PRODUCT CONSTRUCTOR FOR PRODUCT OBJECTS
+function Product(filepath, name) {
+  this.filepath = filepath;
+  this.imgname = imgname;
+  this.numviewed = numviewed;
+  this.numclicked = numclicked;
+  this.imageid = imageid;
+  Product.allProducts.push(this);
+}
+
+
+/////////////////////////////////////////////////////////////////////
+/////////////////////////////////////////////////////////////////////
+
+
+// CREATE FUNCTION FOR DETERMINING WHETHER ARRAY
+// ELEMENTS MATCH EACH OTHER
+// DO NOT DISPLAY MATCHING ELEMENTS
+// RUN THIS THREE TIMES AND RETURN THREE
+// DIFFERENT ARRAY ELEMENTS
+
+
+// COMPARE VALUES WITHIN ARRAY TO OTHER VALUES WITHIN ARRAY
+var randIndex = [];
+
+function randomProd(){
+  var randomIndex = Math.floor(Math.random () * Product.allProducts.length);
+  randIndex.push(randomProd); 
+}
+
+var begRandArray[0] = 0;
+var begRandArray[1] = 0;
+var begRandArray[2] = 0;
+
```

Comment:
These statements are invalid.  You could decalare the begRandArray as an array with the var keyword, and then address it's elements without it.